src/cinder/Signals.cpp

```diff
@@ -111,6 +111,22 @@ ScopedConnection& ScopedConnection::operator=( ScopedConnection &&rhs )
 	return *this;
 }
 
+ConnectionList::~ConnectionList()
+{
+	clear();
+}
+
+void ConnectionList::add( Connection &&target )
+{
+	mConnections.emplace_back( target );
+}
+
+void ConnectionList::clear()
+{
+	for( auto &conn : mConnections )
```

Comment:
I don't have a dev-environment at hand right now, but I think there is a cleaner solution: Imho move constructor and move assignment operator should properly reset the object variable you move from to a null state, such that a call to `disconnect` on a moved from object simply doesn't have any effect. 

Do those problems only affect VS2013? Because I'd expected the compiler to automatically create a move constructor for `Connection` that has exactly that effect, but VS2013 seems to be pretty limited in that regard.